identity/README.md

```diff
@@ -31,7 +31,7 @@ So the required steps are:
 
 * configure and run nginx (see `/nginx`)
 * update your hosts file (also described in `/nginx`)
-* make sure that your frontend.properties file contains
+* make sure that your `~/.gu/frontend.conf` file contains
 
```

Comment:
talked in person, but for future reference, no because it's not an override, it's just the normal parameter that everyone needs, so it should be in the config file that's in S3